python/edgar_filing_searcher/parsers/main.py

```diff
@@ -39,13 +42,17 @@ def main():
             accession_no=accession_no,
             cik_no=cik,
             filing_date=filing_date)
-        session.add(company_row)
-        session.add(edgar_filing_row)
+
+        app = create_app()
```

Comment:
you could extract this into a function called setup_db_connection() so that future you understands what this is doing.